controllers/imagepolicy_controller.go

```diff
@@ -164,3 +183,28 @@ func (r *ImagePolicyReconciler) imagePoliciesForRepository(obj handler.MapObject
 	}
 	return reqs
 }
+
+// event emits a Kubernetes event and forwards the event to notification controller if configured
+func (r *ImagePolicyReconciler) event(policy imagev1alpha1.ImagePolicy, severity, msg string) {
+	if r.EventRecorder != nil {
+		r.EventRecorder.Eventf(&policy, "Normal", severity, msg)
```

Comment:
Why use `Eventf`?

Reply:
As you can imagine, I did a lot of copy-pasting :smile: